### DISCUSSION TITLE
Static IP address

### DISCUSSION THREAD

**Discussion by danielnowrot** [nextcloud/all-in-one#2963] [Questions] [answered]
Hi!
Where I can find usefull information about my question: how to set static IP address?
I mean in Nextcloud AIO. 
Someone knows?

**Comment by szaimen** [comment, marked as answer]
This is unrelated to AIO and should be asked in an appropriate Linux forum.

**Reply by danielnowrot** [reply]
Ok, thats mean it's not possibble from AIO GUI
